source/renderer/app/components/staking/stake-pools/StakePoolsTable.tsx

```diff
@@ -142,277 +42,213 @@ type Props = {
   onSelect?: (...args: Array<any>) => any;
   containerClassName: string;
   numberOfRankedStakePools: number;
-  selectedPoolId?: number | null | undefined;
+  selectedPoolId?: string;
   onOpenExternalLink: (...args: Array<any>) => any;
   currentLocale: string;
   onTableHeaderMouseEnter: (...args: Array<any>) => any;
   onTableHeaderMouseLeave: (...args: Array<any>) => any;
+  intl: Intl;
 };
 type State = {
   isPreloading: boolean;
-  stakePoolsOrder: string;
-  stakePoolsSortBy: string;
+  stakePoolsOrder: StakePoolsOrder;
+  stakePoolsSortBy: keyof StakePool;
 };
-const initialState = {
+const initialState: State = {
   isPreloading: true,
-  stakePoolsOrder: 'asc',
+  stakePoolsOrder: StakePoolsOrder.Asc,
```

Comment:
```suggestion
  stakePoolsOrder: defaultTableOrdering.ranking,
```

Reply:
This is not related to `ranking` sort order.